fix: Update Shoot creation guide in light of new Monitoring tab (#280)
citynetwork/docs: docs/howto/kubernetes/gardener/create-shoot-cluster.md

If necessary, be sure to request a quota increase via our [{{support}}](https://{{support_domain}}/servicedesk).

## Viewing details and monitoring

After the new shoot cluster finishes bootstrapping, you may click on its row to bring all relevant details into view.
For instance, click the *Details* tab to get networking information about the worker nodes.

![Networking information about the worker nodes](assets/create-shoot-09.png)

Next, go to the *Monitoring* tab.

> There, you will notice two orange buttons, each for launching [*Prometheus*](https://prometheus.io) or [*Plutono*](https://github.com/credativ/plutono).

Reviewer: The rest of the guide uses weak emphasis (italics) for UI labels. Since we have no tab or button labeled, literally, "Prometheus" or "Plutono", I'd suggest dropping the italics here:

```suggestion
There, you will notice two orange buttons, each for launching [Prometheus](https://prometheus.io) or [Plutono](https://github.com/credativ/plutono).
```

Reviewer: On second thought, there's another modification I'd suggest:

```suggestion
There, you will notice two orange buttons: one for launching [Prometheus](https://prometheus.io), and one for [Plutono](https://github.com/credativ/plutono).
```

Author: Ah, yes, better.

![Monitoring tab for launching Prometheus or Plutono](assets/create-shoot-10.png)

As an example, we have clicked the button for Plutono.

> Before the dashboard launches a pop-up window appears, offering the default username (`admin`) and the corresponding automatically generated password (not shown by default).

Reviewer: Using "offering" here sounds weird to me.
```suggestion
Prior to launching the dashboard, a pop-up window appears.
This shows the default username (`admin`) and the masked generated password.
```
